javascript-modal/styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+#trigger {
+  width: 12vw;
+  height: 5vh;
+  border: none;
+  border-radius: 7px;
+  font-size: 1.5rem;
+  color: #fff;
+  background-color: #4a4ae9;
+  box-shadow: 0 2px 4px black;
+}
+
+#trigger:hover {
+  cursor: pointer;
+}
+
+#trigger:focus {
+  display: none;
+}
+
+.modal-content {
+  text-align: center;
+  font-weight: bold;
+  font-size: 1.1rem;
+  position: fixed;
+  width: 25vw;
+  height: 25vh;
+  padding-top: 1rem;
+  top: 25%;
+  left: 35%;
+  display: none;
+  background-color: #fff;
+  z-index: 100;
+}
+
+.modal-content.off {
+  display: block;
+}
+
+.no {
+  width: 8vw;
+  height: 5vh;
```

Comment:
Use `px` instead of `vw` and `vh` for almost everything. `vh` is rare, and `vw` is almost never used.